app/src/features/alert/banner/SystemAlertBanner.tsx

```diff
@@ -0,0 +1,73 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Collapse from '@mui/material/Collapse';
+import grey from '@mui/material/colors/grey';
+import AlertBar from 'components/alert/AlertBar';
+import dayjs from 'dayjs';
+import { useBiohubApi } from 'hooks/useBioHubApi';
+import useDataLoader from 'hooks/useDataLoader';
+import { useEffect, useState } from 'react';
+import { TransitionGroup } from 'react-transition-group';
+
+export enum SystemAlertBannerEnum {
```

Comment:
I would maybe move this into `useAlertApi.interface.ts`.

The filter param `IAlertFilterParams.types` is just defined as a `string[]`, but it looks like its value should be an array of these strings.

Reply:
Thanks, fixed.